src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -14,175 +14,25 @@
-        runner.addPlugin(TestManagerResultsPlugin('ExportToFile', stmReportPath));
-        stmResultsPluginAddedToRunner = true;
-    end
-end
-
-% Produce PDF test report (Not supported on MacOS platforms and below R2017a)
-if ~isempty(pdfReportPath)
-    if ismac
-        warning('MATLAB:testArtifact:unSupportedPlatform', ...
-            'Producing a PDF test report is not currently supported on MacOS platforms.');
-    elseif ~testReportPluginPresent
-        issuePDFReportUnsupportedWarning;
-    else
-        preparePath(pdfReportPath);
-        import('matlab.unittest.plugins.TestReportPlugin');
-        runner.addPlugin(TestReportPlugin.producingPDF(pdfReportPath));
-        
-        if ~stmResultsPluginAddedToRunner && stmResultsPluginPresent
-            runner.addPlugin(TestManagerResultsPlugin);
-        end
-    end
-end
-
-results = runner.run(suite);
-failed = any([results.Failed]);
-
-function preparePath(path)
-dir = fileparts(path);
-dirExists = isempty(dir) || exist(dir,'dir') == 7;
-if ~dirExists		
-    mkdir(dir);		    
-end
-
-function tapFile = getTapResultFile(resultsDir)
-import('matlab.unittest.plugins.ToFile');
-preparePath(resultsDir);
-fclose(fopen(resultsDir,'w'));
-tapFile = matlab.unittest.plugins.ToFile(resultsDir);
-
-function suite = getTestSuite()
-import('matlab.unittest.TestSuite');
-BASE_VERSION_TESTSUITE_SUPPORT = '9.0';
-if verLessThan('matlab',BASE_VERSION_TESTSUITE_SUPPORT)
-    suite = matlab.unittest.TestSuite.fromFolder(pwd,'IncludingSubfolders',true);
-else
-    suite = testsuite(pwd,'IncludeSubfolders',true);
-end
-
-function plugin = CoberturaFormat(varargin)
-plugin = matlab.unittest.plugins.codecoverage.CoberturaFormat(varargin{:});
-
-function plugin = TestManagerResultsPlugin(varargin)
-plugin = sltest.plugins.TestManagerResultsPlugin(varargin{:});
-
-function tf = testReportPluginPresent
-BASE_VERSION_REPORTPLUGIN_SUPPORT = '9.2'; % R2017a 
-
-tf = ~verLessThan('matlab',BASE_VERSION_REPORTPLUGIN_SUPPORT);
-
-function tf = stmResultsPluginPresent
-tf = logical(exist('sltest.plugins.TestManagerResultsPlugin', 'class'));
-
-function tf = coberturaModelCoverageSupported
-BASE_VERSION_MODELCOVERAGE_SUPPORT = '9.5'; % R2018b
-
-tf = ~verLessThan('matlab',BASE_VERSION_MODELCOVERAGE_SUPPORT);
-
-function tf = exportSTMResultsSupported
-BASE_VERSION_EXPORTSTMRESULTS_SUPPORT = '9.6'; % R2019a
-
-tf = ~verLessThan('matlab',BASE_VERSION_EXPORTSTMRESULTS_SUPPORT);
-
-function issuePDFReportUnsupportedWarning
-warning('MATLAB:testArtifact:pdfReportNotSupported', ...
-    'Producing a test report in PDF format is not supported in the current MATLAB release.');
-
-function issueExportSTMResultsUnsupportedWarning
-warning('MATLAB:testArtifact:cannotExportSimulinkTestManagerResults', ...
-    ['Unable to export Simulink Test Manager results. This feature ', ...
-    'requires a Simulink Test license and is supported only in MATLAB R2019a or a newer release.']);
+srcFldr = getenv('WORKSPACE');
```

Comment:
You should not need to specify `SourceFolder` to achieve the current Jenkins Plugin behavior.

Reply:
Done.